tests/utils.go

```diff
@@ -763,6 +764,10 @@ func BeforeTestSuitSetup() {
 	CreatePVC(osWindows, defaultWindowsDiskSize, Config.StorageClassWindows)
 	CreatePVC(osRhel, defaultRhelDiskSize, Config.StorageClassRhel)
 
+	if IsRunningOnKindInfraIPv6() {
```

Comment:
This issue happens in all kind based providers, as stated in the [PR's description](https://github.com/kubevirt/kubevirt/pull/3374#issue-413443532).

As such, I think the pod preset should be created on **all providers based on kind infra**, not on kind ipv6 provider.

Reply:
lets see please what team decide
Alona and Vladik in the current groom said its ok to take it afterwards.
one reason is that if kind has running already migration on other lanes it would failed if indeed it would already need it

i must focus on my tasks first to wrap them

Reply:
@oshoval , lets limit the discussion on the technical aspects of the review.

The reasoning of not failing for other kind tests is irrelevant, it may be the case that no migration tests are running on kind except the ipv6 ones.

You could argue that we want to first pass the IPv6 tests due to the urgency of the matter and therefore the **correct** solution is therefore better differed to a following PR.